From 3a40b6785b74dba464eeb62a09d30aa8b2626530 Mon Sep 17 00:00:00 2001
From: Qi Yadong <yadong.qi@intel.com>
Date: Mon, 7 May 2018 16:39:53 +0800
Subject: [PATCH 0932/1240] BUG fix: add break in case IC_PM_GET_CPU_STATE

Add break in case IC_PM_GET_CPU_STATE to prevent it
fallthrough to next case.

Change-Id: Ibebcfa2c4908b479edaa12b8a83299c626b17919
Signed-off-by: Qi Yadong <yadong.qi@intel.com>
---
 drivers/char/vhm/vhm_dev.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/char/vhm/vhm_dev.c b/drivers/char/vhm/vhm_dev.c
index 069cdb9..025e451 100644
--- a/drivers/char/vhm/vhm_dev.c
+++ b/drivers/char/vhm/vhm_dev.c
@@ -578,6 +578,8 @@ static long vhm_dev_ioctl(struct file *filep,
 			ret = -EFAULT;
 			break;
 		}
+
+		break;
 	}
 
 	default:
-- 
2.7.4

